.travis.yml.jinja

```diff
@@ -0,0 +1,82 @@
+language: python
+cache:
+  directories:
+    - $HOME/.cache
```

Comment:
```suggestion
    - $HOME/.cache
    - $HOME/.cache/pre-commit
```

Reply:
Ain't it better to cache all cache?

Reply:
Ah yes, caching all the cache directory. I don't know if that could be too aggressive. We usually have .cache/pip and .cache/pre-commit.

Reply:
OK I was wondering why wasn't done like this, but nevermind, you're right, I'll better leave it as it was.